source/MRMesh/MRContoursCut.h

```diff
@@ -57,15 +58,23 @@ MRMESH_API void subdivideLoneContours( Mesh& mesh, const OneMeshContours& contou
 MRMESH_API OneMeshContours getOneMeshIntersectionContours( const Mesh& meshA, const Mesh& meshB, const ContinuousContours& contours, bool getMeshAIntersections,
     const CoordinateConverters& converters, const AffineXf3f* rigidB2A = nullptr );
 
+
+struct SearchPathSettings
+{
+    GeodesicPathApprox geodesicPathApprox = GeodesicPathApprox::DijkstraAStar;
+    int maxReduceIters{ 0 };
```

Comment:
0 is not good default value (better 5)

Reply:
value 5 default for reducePath, but for convertMeshTriPointsToMeshContour default is 0